php/src/Google/Protobuf/Internal/DescriptorProto.php

```diff
@@ -119,11 +129,6 @@ public function setName($var)
         return $this;
     }
 
-    public function hasName()
```

Comment:
Removing this is technical a backward incompatible change. However since the method appears to be useless (has_name is never true AFAICS?) this will hopefully not be disruptive.

Reply:
The situation is that has method has already been generated generally.
Generating has methods for descriptors is a duplicate.

Reply:
I think has methods are not generated for repeated fields, and should not be.

So this is backward incompatible, but hopefully rarely used. Anyone who was using it should check the repeated field size instead.